notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -4,7 +4,7 @@
       "cell_type": "code",
```

Comment:
<p><strong>If you don't know your project ID</strong>, you can get it using the&nbsp;<code>gcloud</code>&nbsp;CLI.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>